multichain-testing/scripts/install-bundle.ts

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env -S node --import ts-blank-space/register
+import '@endo/init/debug.js';
+
+import {
+  installBundle,
+  txFlags,
+} from '@agoric/deploy-script-support/src/permissioned-deployment.js';
+import { makeCmdRunner, makeFileRd } from '@agoric/pola-io';
+import { execa } from 'execa';
+import { toCLIOptions } from '@agoric/internal';
+import { fetchEnvNetworkConfig } from '@agoric/client-utils';
+
+const Usage = '_script_ bundle.json';
+
+const main = async (
```

Comment:
I'm surprised we need a one-off for this basic functionality

Reply:
yeah; not clear that it pulls its weight.

I eventually had to figure out how to call `agd` straight from the `Makefile`; might as well do `agd tx swingset install-bundle ...`, I suppose.